BlackJack.py

```diff
@@ -0,0 +1,235 @@
+from random import choice
+from aenum import Enum, NoAlias
+from UserInputProvider import *
+
+
+class Color(Enum):
+    HEART = 1
+    DIAMOND = 2
+    SPADE = 3
+    CLUB = 4
+
+    @classmethod
+    def values(cls):
+        return list(cls)
+
+
+class Sign(Enum):
+    _settings_ = NoAlias
+    ACE = 1
+    TWO = 2
+    THREE = 3
+    FOUR = 4
+    FIVE = 5
+    SIX = 6
+    SEVEN = 7
+    EIGHT = 8
+    NINE = 9
+    TEN = 10
+    JACK = 10
+    QUEEN = 10
+    KING = 10
+
+    @classmethod
+    def values(cls):
+        return list(cls)
+
+
+class Card(object):
+
+    def __init__(self, sign, color):
+        self.sign = sign
+        self.color = color
+
+    def __repr__(self):
+        return self.sign.name + " " + self.color.name
+
+
+class Deck(object):
+
+    def __init__(self):
+        self.cards = []
+        for color in Color.values():
+            for sign in Sign.values():
+                self.cards.append(Card(sign, color))
+
+    def shuffled_deck(self):
+        cards = []
+        for i in range(len(self.cards)):
+            card = choice(self.cards)
+            self.delete_card_from_deck(self.cards, card)
+            cards.append(str(card))
+        return cards
+
+    def delete_card_from_deck(self, deck, card):
+        deck.remove(card)
+
+
+class Contestant(object):
+
+    def __init__(self, hand):
+        self.hand = hand
+        self.shuffled_deck = Deck().shuffled_deck()
+
+    def get_cards(self, number):
+        for i in range(number):
+            self.hand.append(self.shuffled_deck[0])
+            Deck().delete_card_from_deck(self.shuffled_deck, self.shuffled_deck[0])
+
+    def count_score(self):
+        result = 0
+        for card in self.hand:
+            for sign in Sign.values():
+                if sign.name == card.split()[0]:
+                    result += sign.value
+        return result
+
+    def check_if_ace_in_hand(self):
```

Comment:
if Sign.ACE in self.hand: - czy to nie bedzie robic dokladnie tego samego co ta funkcja?